packages/dropdowns/src/elements/Menu/Items/HeaderItem.spec.tsx

```diff
@@ -0,0 +1,85 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { render } from 'garden-test-utils';
+import { Dropdown, Trigger, Menu, HeaderItem } from '../../..';
+import { DEFAULT_THEME } from '@zendeskgarden/react-theming';
+
+describe('HeaderItem', () => {
+  it('passes ref to underlying DOM element', () => {
+    const ref = React.createRef<HTMLLIElement>();
+
+    const { getByTestId } = render(
+      <Dropdown>
+        <Trigger>
+          <button data-test-id="trigger">Test</button>
```

Comment:
There are a few unused `data-test-id` in this file.

Reply:
Removed.